src/server/game/World/World.cpp

```diff
@@ -1282,6 +1282,8 @@ void World::LoadConfigSettings(bool reload)
 
     _bool_configs[CONFIG_ALLOWS_RANK_MOD_FOR_PET_HEALTH] = sConfigMgr->GetOption<bool>("Pet.RankMod.Health", true);
 
+    _bool_configs[CONFIG_MUNCHING_BLIZZLIKE] = sConfigMgr->GetOption<bool>("MunchingBlizzlike.Enabled", true);
```

Comment:
As you only use this in one place you could technically move `sConfigMgr->GetOption<bool>("MunchingBlizzlike.Enabled", true)` directly into the if statement and not have it in world files. But it doesn't matter

Reply:
Personally I prefer how it is currently for consistency, but that's just me.

Reply:
My understanding is that every config should be loaded on start up to save up on I/O operations from reading the text file, which are rather resource intensive